tests/api/items/inventory_view.test.js

```diff
@@ -31,6 +33,23 @@ describe('items:inventory-view', () => {
     res.itemsByDate.should.be.an.Array()
   })
 
+  it('should return an inventory-view for user items without shelf', async () => {
+    const user = await createUserWithItems({}, _.times(2, () => Object.new({ listing: 'public' })))
```

Comment:
The test fails because this line throws `TypeError: Object.new is not a function`

```suggestion
    const user = await createUserWithItems({}, _.times(2, () => ({ listing: 'public' })))
```
Explaination: "[In an arrow function] To return an object literal expression requires parentheses around expression" [[source](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#Advanced_syntax)]

or alternatively

```suggestion
    const itemsData = [ { listing: 'public' }, { listing: 'public' } ]
    const user = await createUserWithItems({}, itemsData)
```

Once fixed, the test passes

Reply:
done!